content/guides/tinacms/adding-internationalization-to-tina/modify-get-static-props.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Modify getStaticProps()
+last_edited: '2022-04-06T10:00:00.000Z'
+---
+
+Next, we'll want to update `getStaticProps` to include `locale` as part of the `relativePath`.
+
+```js
+/**
+ * /pages/post/[filename].tsx
+ */
+
+// `locale` is provided alongside `params`
+const getStaticProps = async({ params, locale }) {
```

Comment:
Should we add a try / catch here as [we do in the data fetching section](https://tinacms-site-next-git-feature-add-i18n-guide-tinacms.vercel.app/docs/features/data-fetching/#querying-tina-content-in-nextjs) ?